src/main/java/edu/cornell/kfs/sys/businessobject/lookup/CreateDoneBatchFileLookupableHelperServiceImpl.java

```diff
@@ -16,12 +22,36 @@
 import org.kuali.rice.kim.api.identity.Person;
 
 import edu.cornell.kfs.sys.CUKFSPropertyConstants;
+import edu.cornell.kfs.sys.batch.CreateDoneBatchFile;
 import edu.cornell.kfs.sys.batch.service.CreateDoneBatchFileAuthorizationService;
 
 public class CreateDoneBatchFileLookupableHelperServiceImpl extends BatchFileLookupSearchServiceImpl {
+	private static final Logger LOG = LogManager.getLogger(CreateDoneBatchFileLookupableHelperServiceImpl.class);
 
 	protected CreateDoneBatchFileAuthorizationService createDoneAuthorizationService;
 
+	@Override
+	public List<BusinessObjectBase> getSearchResults(Class<? extends BusinessObjectBase> businessObjectClass,
+			MultivaluedMap<String, String> fieldValues) {
+
+		List<BusinessObjectBase> results = super.getSearchResults(businessObjectClass, fieldValues);
+		List createDoneBatchFiles = new ArrayList<CreateDoneBatchFile>();
```

Comment:
This list is missing the use of generics. Should it just use BusinessObjectBase as the generic type instead?

Reply:
That throws an error, not sure what the generics should be.

Reply:
What kind of error are you getting? It looks as though setting the list type to BusinessObjectBase instead of CreateDoneBatchFile would work. Or is that causing errors on other lines as well?

Reply:
That will work, I misunderstood your suggestion initially. Thank you.